docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc

```diff
@@ -38,7 +38,9 @@ Refer to:
 * <<elastic-agent-container>>
 * <<running-on-kubernetes-managed-by-fleet>>
 * <<running-on-kubernetes-standalone>>
+* <<running-on-gke-standard-managed-by-fleet>>
 * {eck-ref}/k8s-elastic-agent.html[Run {agent} on ECK] -- for {eck} users
+* <<running-on-gke-standard-managed-by-fleet>>
```

Comment:
this line is mentioned twice in this list (line 41 & 43), is it expected?

Reply:
You are right... 🤦🏼‍♂️ thank you!